entc/integration/integration_test.go

```diff
@@ -680,6 +680,25 @@ func Select(t *testing.T, client *ent.Client) {
 	require.Len(gs, 2)
 	require.Equal(hub.QueryUsers().CountX(ctx), gs[0].UsersCount)
 	require.Equal(lab.QueryUsers().CountX(ctx), gs[1].UsersCount)
+
+	// Select Subquery.
+	t.Log("select subquery")
+	subQuery := sql.SelectExpr(sql.Raw("1"))
+	i, err := client.User.Query().Modify(func(s *sql.Selector) {
+		s.Select("*").From(subQuery.As("s"))
+	}).Int(ctx)
```

Comment:
```suggestion
	i, err := client.User.
		Query().
		Modify(func(s *sql.Selector) {
			subQuery := sql.SelectExpr(sql.Raw("1")).As("s")
			s.Select("*").From(subQuery)
		}).
		Int(ctx)
```

Style preference. I prefer to keep subqueries in the scope they are used (if it's possible), and add extra line breaks if it keeps the code more readable (don't have a strong rule here). I hope you don't mind 😃